src/main/java/at/hannibal2/skyhanni/features/garden/GardenLevelDisplay.kt

```diff
@@ -78,14 +82,19 @@ class GardenLevelDisplay {
     @SubscribeEvent
     fun onInventoryOpen(event: InventoryFullyOpenedEvent) {
         if (!GardenAPI.inGarden()) return
-        if (event.inventoryName != "Desk") return
-        val item = event.inventoryItems[4]!!
-
-        val currentLevel = currentLevelPattern.matchMatcher(item.name.removeColor()) {
-            group("currentLevel").romanToDecimalIfNecessary()
+        val item = when (event.inventoryName) {
+            "Desk" -> event.inventoryItems[4]!!
+            "SkyBlock Menu" -> event.inventoryItems[10]!!
+            else -> null
         } ?: return
```

Comment:
Do not use `!!` ever. This can throw NPEs
```suggestion
            "Desk" -> event.inventoryItems[4] ?: return
            "SkyBlock Menu" -> event.inventoryItems[10] ?: return
            else -> return
        }
```